### PR TITLE
Test Framework refactor fixes

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

**Comment by alexandraorth** [comment]
please retest this please

File: grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java [hunk 1]
```diff
@@ -155,6 +155,15 @@ static void startEmbeddedCassandra() {
         }
     }
 
+    static String randomKeyspace(){
+        if (usingOrientDB()) {
+            return "memory";
```
Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: What? Why does orient get a keyspace called memory?

Comment (alexandraorth) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: I copied this from what we had before, should this not be the case?

Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: No it shouldn't. It should be a random keyspace like any other.

Comment (aelred) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: I'm sure you told me this was a limitation of orientdb. You can already see it in `GraphContext` class.

Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: The URI has to be "memory" for the moment but that is only for unit tests in the factory module.

Comment (alexandraorth) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: Alright, hold off on merging this and I'll push a fix.

**Pull request merged**